source/windows/secure_channel_tls_handler.c

```diff
@@ -1078,7 +1123,7 @@ static int s_do_application_data_decrypt(struct aws_channel_handler *handler) {
 
         SECURITY_STATUS status = DecryptMessage(&sc_handler->sec_handle, &buffer_desc, 0, NULL);
 
-        if (status == SEC_E_OK) {
+        if (status == SEC_E_OK || status == SEC_I_RENEGOTIATE || status == SEC_I_CONTEXT_EXPIRED) {

```

Comment:
I don't know enough to validate this change but given that we have an existing else-if clause for  content expired below here that is unchanged (and now skipped), something feels really off.